straxen/get_corrections.py

```diff
@@ -34,152 +36,75 @@ def correction_options_wrapped(run_id, conf, *arg):
 
     return correction_options_wrapped
 
-
 @export
 @correction_options
-def get_to_pe(run_id, gain_model, n_pmts):
-    if not isinstance(gain_model, tuple):
-        raise ValueError(f"gain_model must be a tuple")
-    if not len(gain_model) == 2:
-        raise ValueError(f"gain_model must have two elements: "
-                         f"the model type and its specific configuration")
-    model_type, model_conf = gain_model
-
-    if model_type == 'disabled':
-        # Somebody messed up
-        raise RuntimeError("Attempt to use a disabled gain model")
-    if model_type == 'CMT_model':
-        if not isinstance(model_conf, tuple) or len(model_conf) != 2:
-            # Raise a value error if the condition is not met. We should have:
-            # ("CMT_model", -> To specify that we want to use the online
-            #                  corrections management tool
-            #   (
-            #   "to_pe_model", -> This is to specify that we want  the gains
-            #   "v1", -> The version of the correction 'v1' is the online version
-            #   )
-            raise ValueError('CMT gain model should be similar to:'
-                             '("CMT_model", ("to_pe_model", "v1"). Instead got:'
-                             f'{model_conf}')
-        # is this the best way to do this?
-        is_nt = n_pmts == straxen.n_tpc_pmts or n_pmts == straxen.n_nveto_pmts or n_pmts == straxen.n_mveto_pmts
-
-        corrections = straxen.CorrectionsManagementServices(is_nt=is_nt)
-        to_pe = corrections.get_corrections_config(run_id, model_conf)
-
-        return to_pe
+def get_correction_from_cmt(run_id, conf):
+    """
+    Get correction from CMT general format is
+    conf = ('correction_name', 'version', True)
+    where True means looking at nT runs, e.g. 
+    get_correction_from_cmt(run_id, conf[:2])
+    special cases:
+    version can be replaced by consant int, float or array
+    when user specify value(s)
+    :param run_id: run id from runDB
+    :param conf: configuration 
+    :return: correction value(s)
+    """
+ 
+    if isinstance(conf, str) and conf.startswith('https://raw'):
```

Comment:
Do you think 1T compatibility check can already handle the is_CMT check [here](https://github.com/ahiguera-mx/straxen/blob/master/straxen/plugins/event_processing.py#L452-L460)? While I would also propose moving this making `map_algo` part to [the decorator function](https://github.com/ahiguera-mx/straxen/blob/master/straxen/get_corrections.py#L27-L32)

Reply:
Might also consider in the 'http://raw' case, return the data after get_resource?

Reply:
You mentioned 1T, that is why I made the comment